src/autoSize.js

```diff
@@ -59,8 +67,8 @@ const imageLoaded = ({ el }) => {
 };
 
 // Returns true if img has sizes attr and the img has loaded.
-const imgCanBeSized = ({ el, existingSizes, elHasAttributes }) => {
-  if (!existingSizes) {
+const imgCanBeSized = ({ el, existingSizes, dataSizes, elHasAttributes }) => {
+  if (!existingSizes && !dataSizes) {
```

Comment:
It _is_ possible for the user to use `ix-sizes` without setting `sizes`. We should only return false here if both are missing.